src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy

```diff
@@ -136,237 +135,188 @@ mode with existing flows and hold flows of different table-mode types"() {
 
         and: "Flow is valid"
         northbound.validateFlow(flow.flowId).each { direction -> assert direction.asExpected }
 
-        when: "Synchronize the flow"
-        with(northbound.synchronizeFlow(flow.flowId)) {
-            !it.rerouted
-        }
-
-        then: "Rules on the switch are reinstalled in single table mode"
-        def flowInfoFromDb2
-        wait(RULES_INSTALLATION_TIME) {
-            flowInfoFromDb2 = database.getFlow(flow.flowId)
-            verifyAll(northbound.getSwitchRules(sw.dpId).flowEntries) { rules ->
-                rules.find { it.cookie == flowInfoFromDb2.forwardPath.cookie.value }.tableId == SINGLE_TABLE_ID
-                rules.find { it.cookie == flowInfoFromDb2.reversePath.cookie.value }.tableId == SINGLE_TABLE_ID
-                //shared rules should be deleted, , issue #3546
-                rules.findAll { it.cookie in sharedRules*.cookie }*.tableId.unique() == [SHARED_RULE_TABLE_ID]
-            }
-        }
-
         when: "Update switch properties(multi_table: true) on the switch"
         northbound.updateSwitchProperties(sw.dpId, changeSwitchPropsMultiTableValue(initSwProps, true))
 
-        then: "Flow rules are still in single table mode on the switch"
-        verifyAll(northbound.getSwitchRules(sw.dpId).flowEntries) { rules ->
-            rules.find { it.cookie == flowInfoFromDb2.forwardPath.cookie.value }.tableId == SINGLE_TABLE_ID
-            rules.find { it.cookie == flowInfoFromDb2.reversePath.cookie.value }.tableId == SINGLE_TABLE_ID
-            //shared rules should be deleted, issue #3546
-            rules.findAll { it.cookie in sharedRules*.cookie }*.tableId.unique() == [SHARED_RULE_TABLE_ID]
-        }
-
-        and: "Flow is valid and UP"
-        northbound.validateFlow(flow.flowId).each { direction -> assert direction.asExpected }
-        wait(WAIT_OFFSET / 2) { assert northbound.getFlowStatus(flow.flowId).status == FlowState.UP }
+        and: "#eventName"
+        event(flow)
 
-        when: "Update the flow"
-        flowHelperV2.updateFlow(flow.flowId, flow.tap { it.description = it.description + " updated" })
-
-        then: "Flow rules on the switch are recreated in multi table mode"
-        def flowInfoFromDb3
+        then: "Flow rules go to multi table mode"
         wait(RULES_INSTALLATION_TIME) {
-            flowInfoFromDb3 = database.getFlow(flow.flowId)
-            verifyAll(northbound.getSwitchRules(sw.dpId).flowEntries) { rules ->
-                rules.find { it.cookie == flowInfoFromDb3.forwardPath.cookie.value }.tableId == INGRESS_RULE_MULTI_TABLE_ID
-                rules.find { it.cookie == flowInfoFromDb3.reversePath.cookie.value }.tableId == INGRESS_RULE_MULTI_TABLE_ID
-                rules.findAll { it.cookie in sharedRules*.cookie }*.tableId.unique() == [SHARED_RULE_TABLE_ID]
-            }
+            "flow rules are in tables"(sw, flow) == [INGRESS_RULE_MULTI_TABLE_ID]
```

Comment:
What about section `and: "Flow is valid and UP"` after updating?
Why it was removed? Flow can be down after update, even if rules are in multi_table_mode

Reply:
Returned this validation.